cypress/e2e/session_timeout.cy.js

```diff
@@ -0,0 +1,74 @@
+import './base.cy'
+
+describe('session timeout tests', () => {
+
+  beforeEach(()=>{
+    cy.clock();
+    cy.start()
+  });
+
+  it('session_timeout_dialog_success', () => {
```

Comment:
This string is supposed to be human-readable when looking at test results to understand what the test does. Here you could use `it('Displays the timeout warning after 15 minutes', ` for instance